.circleci/config.yml

```diff
@@ -294,6 +304,12 @@ jobs:
     environment:
       IMAGES: "dashboard"
     <<: *build_images
+  build_pinniped_proxy:
+    docker:
+      - image: circleci/rust
```

Comment:
Honestly this image does nothing since we are then building in a docker container, I am not sure what would be faster, if reusing the `golang` image or doing this.

Reply:
Ah - ok. I'll switch it to the golang one then.